### PR TITLE
Flip Y Axis

### PR DESCRIPTION
**Pull request by astronouth7303**
Per some discussions on discord, flip the Y axis so that PPB's concept of space & coordinates match those of traditional geometry & algebra.

I think I'm going to write a full blog post explaining this.

### CODE REVIEW

**Comment by astronouth7303** [comment]
Oh, this depends on #204.

File: ppb/camera.py [hunk 1]
```diff
@@ -96,10 +96,18 @@ def in_frame(self, sprite: BaseSprite) -> bool:
...
+        """
         offset = (point - self.viewport_offset) * (1/self.pixel_ratio)
-        return self.position + offset
+        loc = self.position + offset
+        return loc.update(y=-loc.y)
 
-    def translate_to_viewport(self, point:Vector) -> Vector:
+    def translate_to_viewport(self, point: Vector) -> Vector:
+        """
+        Converts a vector from in-game to PyGame coordinate space
```
Comment (pathunstrom) [inline ppb/camera.py]: This is more about screen space, not pygame specific (We'll have  a similar pixel array context no matter which rendering library we use.)

Comment (astronouth7303) [reply ppb/camera.py]: Umm... You don't have too? Taking to discord.

**Pull request merged**